packages/harpstrata/src/get-harp-strata/utils/get-active-ids-pair/get-active-ids-pair.test.ts

```diff
@@ -1,122 +1,26 @@
-import { getDegree, DegreeIds, PitchIds, getPitch } from 'harpparts'
-
-import { EXAMPLE_STRATA } from '../../testResources'
-import type { ActiveIds } from '../../../types'
-import type { IsActiveProps } from '../../../types'
+import { DegreeIds, PitchIds } from 'harpparts'
 
 import { getActiveIdsPair } from './index'
 
-const c = getPitch(PitchIds.C)
-const d = getPitch(PitchIds.D)
-const e = getPitch(PitchIds.E)
-const f = getPitch(PitchIds.F)
-
-const root = getDegree(DegreeIds.Root)
-const second = getDegree(DegreeIds.Second)
-const third = getDegree(DegreeIds.Third)
-const fourth = getDegree(DegreeIds.Fourth)
-
-const basicDegreeMatrix = [
-  [root, second],
-  [third, fourth],
-]
-const basicPitchMatrix = [
-  [c, d],
-  [e, f],
-]
-const baseIsActiveProps: IsActiveProps = {
-  degreeMatrix: basicDegreeMatrix,
-  pitchMatrix: basicPitchMatrix,
-  activeIds: [] as ActiveIds,
-}
-
 test('getActiveIdsPair returns the active ids for a given PitchIds[]', () => {
-  const activePitchIds: ActiveIds = [PitchIds.D, PitchIds.E]
-  const isActiveProps = { ...baseIsActiveProps, activeIds: activePitchIds }
+  const activePitchIds = [PitchIds.D, PitchIds.E] as const
```

Comment:
Perhaps adding some misordered activeIds input tests will be helpful in demonstrating this functions resilience.

Reply:
Done. I don't think this was strictly necessary, but it's put my mind at ease.